src/Vue/doc/index.rst

```diff
@@ -193,6 +193,27 @@ used for all the Vue routes:
     });
 
     app.use(router);
+    
+State Management Library using Pinia (Optional)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you would to use state management **Pinia** (https://pinia.vuejs.org). 
+Install the package via terminal and configure the ``assets/app.js`` file.
+
+.. code-block:: terminal
+
+    $ yarn add pinia
+    $ # or with npm
+    $ npm install pinia
+
+.. code-block:: js
+
+    import { createPinia } from 'pinia'
+    ...
+    document.addEventListener('vue:before-mount', (event) => {
+        app.use(createPinia());
```

Comment:
Currently trying to get this working, but if i follow this part of the documentation the `app` in app.use is undefined.
Should we import the stimulus app here `import { app } from "./bootstrap";`? afaik this should be the Vue app, right?

Reply:
The app should be fetch from `event.detail`

Reply:
Hi @memic84,

I actually revert that commit because I'm thinking to create a separate PR for that part of the docu regarding the optional implementation of Pinia. @Kocal is correct. It should be fetch from `event.detail` I'll update that commit on the future PR.

Here's what it looks like:
```
document.addEventListener('vue:before-mount', (event) => {
        const {
            componentName, // The Vue component's name
            component, // The resolved Vue component
            props, // The props that will be injected to the component
            app, // The Vue application instance
        } = event.detail;
    
        app.use(createPinia());
    });
```
I tried this and it's working :)

Reply:
Or should I include it in this current PR? :) Please let me know. Thank you!